.circleci/config.yml

```diff
@@ -0,0 +1,66 @@
+---
+
+aliases:
+  # common - cache
+  - &cache_restore_bundler
+    restore_cache:
+      name: cache | restore bundle
+      key: v1-gems-{{ arch }}-{{ checksum "Gemfile" }}-{{ checksum "fastlane-plugin-firebase_app_distribution.gemspec" }}
+  - &cache_save_bundler
+    save_cache:
+      name: cache | store bundle
+      key: v1-gems-{{ arch }}-{{ checksum "Gemfile" }}-{{ checksum "fastlane-plugin-firebase_app_distribution.gemspec" }}
+      paths:
+        - "vendor/bundle"
+
+version: 2
+
+jobs:
+  'ruby_2.4.2': &build
+    working_directory: '~/repo'
+    docker:
+      - image: 'circleci/ruby:2.4.2'
+
+    steps:
+      - checkout
+      - *cache_restore_bundler
+      - run:
+          name: deps | bundle install
+          command: |
+            BUNDLER_VERSION=$(awk 'END { print $NF }' Gemfile.lock)
+            gem install bundler
+            bundle check || bundle install --jobs=4 --retry=3 --path vendor/bundle
+      - *cache_save_bundler
+      - run:
+          name: run tests
+          command: bundle exec rake
+      - store_test_results:
+          path: ~/repo/test-results
+      - store_artifacts:
+          path: ~/repo/test-results
+          destination: test-results
+
+  'ruby_2.4.9':
+    <<: *build
+    docker:
+      - image: 'circleci/ruby:2.4.9'
+
+  'ruby_2.5.7':
+    <<: *build
+    docker:
+      - image: 'circleci/ruby:2.5.7'
+
+  'ruby_2.6.5':
+    <<: *build
+    docker:
+      - image: 'circleci/ruby:2.6.5'
+
+workflows:
+  version: 2
+  run_tests:
+    jobs:
+      - 'ruby_2.4.2'
+      - 'ruby_2.4.9'
+      - 'ruby_2.5.7'
+      - 'ruby_2.6.5'
```

Comment:
maybe we should add latest version of ruby?

Reply:
possible, okay

Reply:
done